host-components/src/keep-loader.rs

```diff
@@ -0,0 +1,229 @@
+// SPDX-License-Identifier: Apache-2.0
+#[macro_use]
+extern crate serde_derive;
+
+use ::host_components::*;
+use std::io::prelude::*;
+use std::os::unix::net::{UnixListener, UnixStream};
+use std::process::Command;
+use std::sync::{Arc, Mutex};
+use std::thread;
+
+//IMPORTANT - after every change to this file, the following steps need to be run:
+// remove link file in /etc/systemd/user/
+// recreate link file in /etc/systemd/user/
+// `systemctl daemon-reload`
+/*
```

Comment:
IDK, as mentioned earlier. Either my suggestion works, or this could be:

```console
$ sudo touch -h '/etc/systemd/user/enarx-keep@.service'
$ sudo systemctl daemon-reload
```

Reply:
Don't think this will get fixed in time for the end of the Sprint, so new issue here for all systemd-related questions: https://github.com/enarx/enarx/issues/849